email_messaging_cadence.view.lkml

```diff
@@ -25,51 +25,55 @@ view: email_messaging_cadence {
       (select distinct email_address as click_address,
       message_variation_api_id as c_message_variation_api_id,
       canvas_step_api_id as c_canvas_step_api_id
-      FROM DATALAKE_SHARING.USERS_MESSAGES_EMAIL_CLICK_SHARED)
-
-      SELECT * FROM deliveries
+      FROM DATALAKE_SHARING.USERS_MESSAGES_EMAIL_CLICK_SHARED),
+
+      campaign as
+      ( select id as campaign_id,
+        name as campaign_name,
+        time as updated_timestamp
+      from DATALAKE_SHARING.CHANGELOGS_CAMPAIGN_SHARED
+      ),
```

Comment:
nitpicky syntax thing, but just so it's easier to read let's make all of our ctes look like:

```
campaign as (
    select
        field1,
        field2
    from [source_table]
),
...
```